hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -26,6 +26,12 @@
         },
         controller: function ($http) {
             var ctrl = this;
+
+            $http.get("http://localhost:9874/jmx?qry=Ratis:service=RaftServer,group=*,id=*")
+                .then(function (result){
+                    ctrl.role = result.data.beans[0]
```

Comment:
Do we need this in here? As far as I understand we are only using `ctrl.overview` only in the html file

Reply:
Yes I forgot to remove it !!
thanks

Reply:
https://issues.apache.org/jira/browse/HDDS-6863